sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -541,6 +541,14 @@ object OverwritePartitionsDynamic {
   }
 }
 
+case class DeleteFromTable(
+    child: LogicalPlan,
+    condition: Filter) extends Command {
```

Comment:
I get it that we need the `Filter` operator to resolve subquery, but this is very subtle. Can we add some documents to explain it?